### PR TITLE
Prepare for initial (dev) release

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Housekeeping actions in preparation for an initial release (still an work in progress, but growing more stable)

### CODE REVIEW

File: .github/workflows/R-test-coverage.yml
Comment (jdhoffa) [inline .github/workflows/R-test-coverage.yml:7]: NIT: Is it necessary to specify `master`? I think most/ all of our repositories consistently use `main` at this stage (maybe a good indication, if they don't, to switch them)

File: .github/workflows/R-test-coverage.yml
Comment (jdhoffa) [inline .github/workflows/R-test-coverage.yml:15]: NB: I have noticed that we have both `GITHUB_TOKEN` and `REPO_TOKEN` as organization level secrets. Is there a preference over which to use, for what, and why? Or are they just duplicates.

File: .github/workflows/R-test-coverage.yml
Comment (jdhoffa) [inline .github/workflows/R-test-coverage.yml:1]: Is this something that is a candidate for extraction into the `actions` repo?

File: DESCRIPTION
Comment (jdhoffa) [inline DESCRIPTION:3]: I have a general question to @AlexAxthelm  and @cjyetman: what is our "release" process? 
* Initially, we would only bump `r2dii.*` version numbers once they were accepted to CRAN
* We then entered a situation where we "released" the `pacta.*` packages when they were used to build a "release candidate" for Transition Monitor. This was more or less just to facilitate "easily" re-building the image based on the same package versions, to be able to simulate, e.g. re-running a COP portfolio. 

As far as I can tell, this is now serving neither of the above, but instead the `WebApp` circumstance, so I guess this perhaps is a call to align on our definitions of a "release", otherwise I fear it ends up being a meaningless term. 

NIT: I will extract this into an issue in the `practices` repo.

File: README.md
Comment (jdhoffa) [inline README.md:4]: Do `lifecycle` and `repo-status` serve redundant purposes here? Again, feels like a practice we need to collectively define and understand as a team before using too many badges, otherwise we risk having inconsistent interpretations. 

Relates strongly to: https://github.com/RMI-PACTA/practices/issues/6

**Review by jdhoffa — CHANGES_REQUESTED** [review]
Not strictly blocking these changes, but I think we need to better define our team practices for these changes to really mean something. 

@AlexAxthelm could be a good indication that some of the practices pointed to should be prioritized in this Dev Hangout (also noting that my PR to document our `maintainer` definition hasn't been reviewed by anyone yet).

**Comment by jdhoffa** [comment]
> Housekeeping actions in preparation for an initial release (still an work in progress, but growing more stable)

Ah I didn't read this. In the case that this is "towards" a release, I would suggest leaving the `DESCRIPTION` file at a `x.y.z.9000` version number to indicate it is not yet "released"

File: .github/workflows/R-test-coverage.yml
Comment (AlexAxthelm) [reply .github/workflows/R-test-coverage.yml:15]: IIRC, `REPO_TOPKEN` grants `pull` to our private repos. `GITHUB_TOKEN` is a GitHub builtin that is scoped to the current repo in question.

so REPO_TOKEN is needed if you're pulling in content from another (private) repo.

File: .github/workflows/R-test-coverage.yml
Comment (AlexAxthelm) [reply .github/workflows/R-test-coverage.yml:1]: https://github.com/RMI-PACTA/actions/issues/6

File: .github/workflows/R-test-coverage.yml
Comment (AlexAxthelm) [reply .github/workflows/R-test-coverage.yml:7]: this is leftover from the default R-Lib template. When we switch this repo to using RMI-PACTA/actions, this whole file will be removed, so I'm not worried at this point.

File: DESCRIPTION
Comment (AlexAxthelm) [reply DESCRIPTION:3]: Definitely hear that. My goal with this change is to communicate:
* Not at v1.0 yet (0.*1*.0)
* This is shaped like a release (no dev tag / `.9000`)

File: README.md
Comment (AlexAxthelm) [reply README.md:4]: Hmm. Possibly, but I see them as different? I'll add thoughts in RMI-PACTA/practices#6.

File: DESCRIPTION
Comment (jdhoffa) [reply DESCRIPTION:3]: Indeed. Then I think we have a soup of different "release" strategies and should definitely align on that before we merge a new one into the mix.

File: README.md
Comment (jdhoffa) [reply README.md:4]: Thank you! 

I am genuinely asking, I don't know/ understand the difference. Just hoping to ensure we get the whole team on the same page.

**Review by jdhoffa — APPROVED** [review]
lgtm

**Pull request merged**
